lib/miq_automation_engine/service_models/miq_ae_service_container.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceContainer < MiqAeServiceModelBase
```

Comment:
@kevensen 
From the service model are you planning on exposing its relationships or just the attributes.
With the current implementation only the attributes are going to be exposed and not any relationships.
https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/service_models/miq_ae_service_host.rb#L6 shows how we expose relationships